### PR TITLE
sys-apps/hwinfo: version bump to 21.23

### PR DESCRIPTION
**Pull request by jakeogh**
Includes musl libc support from upstream.

Depends on: https://github.com/gentoo/gentoo/pull/798

### CODE REVIEW

**Comment by monsieurp** [comment]
Make sure the QA checks are all green otherwise we cannot merge the PR.

**Comment by monsieurp** [comment]
@robbat2

**Comment by robbat2** [comment]
@jakeogh please fix the keywords, and squash the manifest into the same commit. (PS if you use repoman it will do some of the checks for you...)

**Comment by jakeogh** [comment]
@robbat2 Thanks, should be correct now.

**Comment by robbat2** [comment]
@jlec already fixed this in bug #565300 today.

**Comment by jlec** [comment]
> @jlec already fixed this in bug #565300 today.

oh, haven't seen this request here.

File: sys-apps/hwinfo/hwinfo-21.23.ebuild [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+src_prepare() {
+	# Respect AR variable.
+	sed -i \
+		-e 's:ar r:$(AR) r:' \
+		src/{,isdn,ids,smp,hd}/Makefile || die
+
+	# Avoid -I directories for dbus because HAL is obsolete.
+	sed -i -e '/CFLAGS/d' src/hd/Makefile || die
```
Comment (jlec) [inline sys-apps/hwinfo/hwinfo-21.23.ebuild]: @jakeogh when you are bumping ebuilds, you need to check if all the code is still necessary, not only that it builds. This sed is superfluous.

Comment (jakeogh) [inline sys-apps/hwinfo/hwinfo-21.23.ebuild]: @jlec thanks, I'll do that. Is it a manual process? My first thought is to hash the file contents before and after by inserting temp checks into the ebuild. looking at gnu sed I see q [exit-code] is available, but that may overcomplicate each sed line, and it's not portable.

**Pull request closed without merging**